aspnet/hosting/directory-structure.rst

```diff
@@ -3,17 +3,42 @@
 Directory Structure
 ===================
 
-By `Sourabh Shirhatti`_
+By `Luke Latham`_
 
 
-In ASP.NET Core, the application directory comprises of three sub-directories. This is unlike previous versions of ASP.NET where the entire application lived inside the web root directory. The recommended permissions for each of the directories are specified in the table below.
+In ASP.NET Core, the application directory, *publish*, is comprised of application files, config files, static assets, packages, and the runtime (for self-contained apps). This is unlike previous versions of ASP.NET, where the entire application lived inside the web root directory.
```

Comment:
I thought it meant that web root == content root, and since those are different now, I thought the statement might be true.

Shall I just delete the line?